docs/core/tools/telemetry.md

```diff
@@ -93,6 +93,20 @@ The telemetry feature collects the following data:
 | >=6.0.300     | If the CLI was invoked from a Continuous Integration environment. For more information, see [Continuous Integration Detection](#continuous-integration-detection).|
 | >=7.0.100     | Hashed PublishAot used for build (MSBuild property) |
 | >=7.0.100     | Hashed PublishProtocol used for build (MSBuild property) |
+| >=8.0.100     | Hashed TargetPlatformIdentifier used for build (MSBuild property) |
+| >=8.0.100     | Hashed RuntimeIdentifier used for build (MSBuild property) |
```

Comment:
```suggestion
```
This property seems already listed on line 76

Reply:
Good catch, this is updated in 0b43ccf9258b99a9bfc0a057e7ef3b037185ef3e